app/src/main/java/com/github/se/polyfit/ui/screen/GraphScreen.kt

```diff
@@ -0,0 +1,99 @@
+package com.github.se.polyfit.ui.screen
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import co.yml.charts.ui.linechart.LineChart
+import com.github.se.polyfit.ui.components.DropDownMenu
+import com.github.se.polyfit.ui.components.lineChartData
+import com.github.se.polyfit.ui.viewModel.GraphViewModel
+import com.github.se.polyfit.ui.viewModel.SortPoints
+
+@Composable
+fun FullGraphScreen() {
+  val isTestEnvironment = System.getProperty("isTestEnvironment") == "true"
+  val isExpanded = remember { mutableStateOf(false) }
+
+  val sortedBy = remember { mutableStateOf("KCAL") }
+
+  val viewModel = viewModel<GraphViewModel>()
```

Comment:
To keep things consistent across the app, it would be great if you used Hilt DI to inject the ViewModel.

Reply:
Alright great idea!